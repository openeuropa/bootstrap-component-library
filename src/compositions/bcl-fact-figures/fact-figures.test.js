import { renderTwigFileAsNode } from "@openeuropa/bcl-test-utils";
import demoData from "@openeuropa/bcl-fact-figures/data/data.js";
const template = "@oe-bcl/bcl-fact-figures/fact-figures.html.twig";

const render = (params) => renderTwigFileAsNode(template, params);

describe("OE - Fact and figures", () => {
  test(`renders correctly in 3 columns`, () => {
    expect.assertions(1);
    return expect(render(demoData)).resolves.toMatchSnapshot();
  });

  test(`renders correctly in 2 columns`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, responsive_columns: 2 })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly in 1 column`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, responsive_columns: 1 })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly with block variant`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, variant: "block" })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly in 3 columns with custom title tag`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, title_tag: "h6" })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly in 2 columns with custom title tag`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, responsive_columns: 2, title_tag: "h6" })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly in 1 column with custom title tag`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, responsive_columns: 1, title_tag: "h6" })
    ).resolves.toMatchSnapshot();
  });

  test(`renders correctly with block variant and with custom title tag`, () => {
    expect.assertions(1);
    return expect(
      render({ ...demoData, variant: "block", title_tag: "h6" })
    ).resolves.toMatchSnapshot();
  });
});
