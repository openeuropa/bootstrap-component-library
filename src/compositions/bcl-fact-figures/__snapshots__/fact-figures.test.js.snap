// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OE - Fact and figures renders correctly in 1 column 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h2
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h2>
    <div>
      <div
        class="row-cols-md-1 row row-cols-1 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly in 1 column with custom title tag 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h6
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h6>
    <div>
      <div
        class="row-cols-md-1 row row-cols-1 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly in 2 columns 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h2
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h2>
    <div>
      <div
        class="row-cols-md-2 row row-cols-1 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly in 2 columns with custom title tag 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h6
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h6>
    <div>
      <div
        class="row-cols-md-2 row row-cols-1 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly in 3 columns 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h2
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h2>
    <div>
      <div
        class="row-cols-md-3 row row-cols-1 row-cols-sm-2 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly in 3 columns with custom title tag 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--default"
  >
    <h6
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h6>
    <div>
      <div
        class="row-cols-md-3 row row-cols-1 row-cols-sm-2 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly with block variant 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--block"
  >
    <h2
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h2>
    <div
      class="bg-light p-3-5 pt-0"
    >
      <div
        class="row-cols-md-3 row row-cols-1 row-cols-sm-2 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;

exports[`OE - Fact and figures renders correctly with block variant and with custom title tag 1`] = `
<jest>
  <div
    class="bcl-fact-figures bcl-fact-figures--block"
  >
    <h6
      class="mb-4 bcl-heading"
    >
      Fact and figures block
    </h6>
    <div
      class="bg-light p-3-5 pt-0"
    >
      <div
        class="row-cols-md-3 row row-cols-1 row-cols-sm-2 g-4 mt-3"
      >
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#sticky"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            1529 JIRA Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira Tickets
          </div>
          <p>
            Nunc condimentum sapien ut nibh finibus suscipit vitae at justo. Morbi quis odio faucibus, commodo tortor id, elementum libero.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inboxes"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            337 Features
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Feature tickets
          </div>
          <p>
            Turpis varius congue venenatis, erat dui feugiat felis.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#shield-check"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            107 Tests
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test tickets
          </div>
          <p>
            Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#file-earmark-code"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            5670 Variants
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Test variants
          </div>
          <p>
            Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#inbox"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            345 Dev Ticket
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Jira ticket
          </div>
          <p>
            Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.
          </p>
        </div>
        <div
          class="col h-100"
        >
          <svg
            class="mb-2 bi icon--l"
          >
            <use
              xlink:href="/icons.svg#eye"
            />
          </svg>
          <div
            class="fs-4 fw-medium"
          >
            43 Components
          </div>
          <div
            class="fs-5 fw-medium"
          >
            Figma components
          </div>
          <p>
            Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.
          </p>
        </div>
      </div>
      <div
        class="border-top d-block pt-3 mt-3"
      >
        <a
          href="/example"
        >
          Read more
        </a>
      </div>
    </div>
  </div>
</jest>
`;
